spacy/tests/test_gold.py

```diff
@@ -298,7 +296,7 @@ def test_gold_biluo_one_to_many(en_vocab, en_tokenizer):
     gold_words = ["I", "flew", "to", "San", "Francisco", "Valley", "."]
     example = Example.from_dict(doc, {"words": gold_words, "entities": entities})
     ner_tags = example.get_aligned_ner()
-    assert ner_tags == ["O", None, "U-LOC", "O"]
+    assert ner_tags == ["O", "U-ORG", "U-LOC", "O"]
 
```

Comment:
The second "word" is "flew to", of which only "flew" is annotated as "ORG", so isn't `None` more appropriate for a mixture of "ORG" and "O" ?

Reply:
Updated: the `predicted` token "Mrs. Smith" consists of "Mrs." being annotated as a PERSON and "Smith" as "O", so shouldn't the final result be None instead?